src/witan/app/util.clj

```diff
@@ -34,6 +34,15 @@
                params)))
       true)))
 
+(defn is-a-number? [txt]
```

Comment:
`read-string` to parse numbers is a probably a bad idea. 

`(is-a-number? "08")` => false

Prefer `(Long/valueOf txt)` and still catch the exception.